helm-charts/0.26.1/charts/spire/templates/spire-agent-config-map.yaml

```diff
@@ -18,39 +18,51 @@ metadata:
   namespace: {{ .Values.global.spire.namespace }}
 data:
   agent.conf: |
-    agent {
-      data_dir = "/run/spire"
-      log_level = {{ .Values.global.spire.logLevel | quote }}
-      server_address = {{ .Values.global.spire.serverAddress | quote }}
-      server_port = {{ .Values.global.spire.serverPort | quote }}
-      socket_path = "/run/spire/sockets/agent.sock"
-      trust_bundle_path = "/run/spire/bundle/bundle.crt"
-      trust_domain = {{ .Values.global.spire.trustDomain | quote }}
-    }
-
-    health_checks {
-      bind_address = "0.0.0.0"
-      bind_port = "9982"
-      listener_enabled = true
-      live_path = "/live"
-      ready_path = "/ready"
-    }
-
-    plugins {
-      NodeAttestor "k8s_psat" {
-        plugin_data {
-          cluster = "vsecm-cluster"
-        }
-      }
-
-      KeyManager "memory" {
-        plugin_data {
-        }
-      }
-
-      WorkloadAttestor "k8s" {
-        plugin_data {
-          skip_kubelet_verification = true
-        }
+    {
```

Comment:
`helm-charts-hardened` uses JSON as the config map format, so I changed the file to a JSON file to align with them.